tests/transports-governance/test_usage_tracking.py

```
@@ -0,0 +1,1061 @@
+            "rate_limit": {"request_max_limit": 10, "request_reset_duration": "1m"},
+            "budget": {"max_limit": 10000, "reset_duration": "1h"},
+        }
+        create_response = governance_client.create_virtual_key(vk_data)
+        assert_response_success(create_response, 201)
+        concurrent_vk = create_response.json()["virtual_key"]
+        cleanup_tracker.add_virtual_key(concurrent_vk["id"])
+
+        messages = [{"role": "user", "content": "Hello"}]
+        headers = {"x-bf-vk": concurrent_vk["value"]}
+
+        def make_request(index):
```

Ruff (0.12.2)

Missing return type annotation for private function `make_request`

(ANN202)